Blog/templates/base_generic.html

```diff
@@ -57,5 +70,11 @@
 </div>
 <script src="https://code.jquery.com/jquery-3.3.1.min.js"></script>
 <script src="https://cdn.metroui.org.ua/v4/js/metro.min.js"></script>
+<script type="application/javascript">
```

Comment:
In what way is this change related to the commenting feature? If it is not related, it should be put into a separate comment where the commit message must contain the reason for this change.